src/backend/src/entities/player.entity.ts

```diff
@@ -2,26 +2,34 @@ import {
   Entity,
   Column,
   PrimaryGeneratedColumn,
-  OneToOne,
-  JoinColumn,
   ManyToOne,
   BaseEntity,
 } from 'typeorm';
 import { Game } from './game.entity';
-import { User } from './user.entity';
+
+// IDK, just placeholder values for now
+export enum EmotionIdEnum {
+  Calm,
+  Greedy,
+  NotGreedy,
+}
```

Comment:
i may be confused about how this is working, but should your enum values also have values attached to them. Like Calm = 1 or something

Reply:
I believe Typescript auto increments values for each based on this

https://www.typescriptlang.org/docs/handbook/enums.html#numeric-enums

Reply:
awesome